app/pb/communities.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package org.couchers.api.communities;
+
+import "google/protobuf/timestamp.proto";
+
+import "pb/api.proto";
+
+message Point {
+  double lat = 1;
+  double lng = 2;
+}
+
+message Node {
+  int32 node_id = 1;
```

Comment:
All other IDs are uint64s. I remember you said this shouldn't be the case, but maybe changing it should be it's own PR.